src/source/Signaling/LwsApiCalls.c

```diff
@@ -1982,15 +2176,16 @@ STATUS receiveLwsMessage(PSignalingClient pSignalingClient, PCHAR pMessage, UINT
 
             // Iterate the state machinery
             CHK_STATUS(signalingStateMachineIterator(pSignalingClient, SIGNALING_GET_CURRENT_TIME(pSignalingClient) + SIGNALING_CONNECT_STATE_TIMEOUT,
-                                                     SIGNALING_STATE_CONNECTED));
+                                                     pSignalingClient->mediaStorageConfig.storageStatus ? SIGNALING_STATE_JOIN_SESSION_CONNECTED
+                                                                                                        : SIGNALING_STATE_CONNECTED));
 
             CHK(FALSE, retStatus);
             break;
 
         case SIGNALING_MESSAGE_TYPE_OFFER:
-            CHK(pSignalingMessageWrapper->receivedSignalingMessage.signalingMessage.peerClientId[0] != '\0',
-                STATUS_SIGNALING_NO_PEER_CLIENT_ID_IN_MESSAGE);
-            // Explicit fall-through !!!
+            // CHK(pSignalingMessageWrapper->receivedSignalingMessage.signalingMessage.peerClientId[0] != '\0',
```

Comment:
Nit: Remove comment. Also, we are not expecting a Peer client ID in the message? Is this specific to storage? What about non storage use cases?

Reply:
It is specific to storage.